docs/Documentation/Drivers/Image/Params.md

```diff
@@ -1,145 +1,444 @@
+      // global attributes:
+      		:description = "VIC parameter file" ;
+      		:history = "Created: Mon Mar 28 09:32:46 2016\n/Users/jhamman/anaconda/lib/python3.4/site-packages/ipykernel/__main__.py -f /Users/jhamman/Library/Jupyter/runtime/kernel-b8627880-fe9e-4d1a-b2d7-fa3fb6a23e22.json\n" ;
+      		:source = "/Users/jhamman/anaconda/lib/python3.4/site-packages/ipykernel/__main__.py" ;
+      		:username = "jhamman" ;
+      		:host = "Joes-iMac.local" ;
+      }
 ```
 
-Add this library path to your `$PYTHONPATH` in your `.bashrc`:
-
-```shell
-export PYTHONPATH=$HOME/lib/python:$PYTHONPATH
-```
+# Soil Parameters
+
+The Soil Parameters serve three main purposes:
+
+*   Define the cell ID number of each grid cell. This ID number is essentially a database key that links a grid cell to its parameters in the various parameter files.
+*   Define the grid cell soil parameters
+*   Define initial soil moisture conditions, to be used in the absence of an initial state file.
+
+The soil parameters are supplied to VIC in a NetCDF file, with a separate variable for each soil parameter.
+
+Below is a list of soil parameters.
+
+| Column                            | Variable Name             | Units     | Number of Values  | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+|--------------------------------   |------------------------   |---------- |------------------ |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------    |
+| 1                                 | run_cell                  | N/A       | 1                 | 1 = Run Grid Cell, 0 = Do Not Run                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Since we don't have "columns" in these parameter files, can we update to show dimensions?  see @yixinmao's state file page for an example of what I'm thinking: http://vic.readthedocs.io/en/develop/Documentation/Drivers/Image/StateFile/#netcdf-state-file-variables-state-variables